apps/emqx_authn/src/emqx_authn.erl

```diff
@@ -126,3 +134,37 @@ get_enabled_authns() ->
 
 tally_authenticators(#{id := AuthenticatorName}, Acc) ->
     maps:update_with(AuthenticatorName, fun(N) -> N + 1 end, 1, Acc).
+
+%%------------------------------------------------------------------------------
+%% Data backup
+%%------------------------------------------------------------------------------
+
+-define(IMPORT_OPTS, #{rawconf_with_defaults => true, override_to => cluster}).
+
+import_data(RawConf) ->
+    AuthnList = maps:get(<<"authentication">>, RawConf, []),
+    lists:foreach(
+        fun(Authn) ->
+            case update_conf(update_req(Authn)) of
+                %% Assume not_found error
+                {error, _} ->
+                    {ok, _} = update_conf(create_req(Authn));
+                {ok, _} ->
+                    ok
+            end
+        end,
+        AuthnList
+    ).
```

Comment:
we need support delete action, for deleting authn not exist in rawConf.

Reply:
Note: it has been agreed to merge imported data with the existing one (without deleting) in the first iteration to keep it consistent with EMQX4.4 data import behaviour (https://emqx.atlassian.net/browse/EMQX-9203?focusedCommentId=28667). 
Replacing (restore backup) may be implemented later, as it was discussed on the demo meeting.